views/error.html

```diff
@@ -1,8 +1,11 @@
-<h1 class="base--h1">I'm error!</h1>
+<div class="not-found">
```

Comment:
Why did you add an `error.html` view instead of re-using the `404` view directly?